content/03.text-mining.md

```diff
@@ -15,35 +15,59 @@
+Tudor et al., used sentence simplification within their extraction pipeline to detecting protein phosphorylation events [@doi:10.1093/database/bav020].
+The sentence simplifier broke complex sentences that contain multiple protein events into smaller sentences that contain only one distinct event.
+By breaking these sentences down the authors were able to increase their recall performance.
+However, some sentences were too complex for the simplifier to handle which results in errors being generated [@doi:10.1093/database/bav020].
+These errors highlight a crucial need for future algorithms to be generalizable enough to handle various forms of complex sentences.
 
-Other studies focused on pattern matching to identify sentences.
-Some patterns were constructed to match sentences based on their part of speech tags and keywords [@doi:10.1093/nar/gkx462; @doi:10.1186/s12859-018-2103-8; @doi:10.1371/journal.pone.0152725; @doi:10.1093/bioinformatics/btg449; @doi:10.1371/journal.pone.0152725; @doi:10.1186/1471-2105-14-181].
-Besides using part of speech tags, some approaches used dependency path trees to construct patterns [@doi:10.1016/j.jbi.2015.08.008; @doi:10.1093/database/bay108].
-Dependency path trees are data structures that depict how words modify one another (example shown in Figure {@fig:dependency-parse-tree-example}).
-Approaches that used these trees achieved similar detection performance methods that focused on part of speech tags.
-Similar to sentence simplification, these methods require a significant amount of manual effort and expert knowledge to work well.
-One future direction is to find a way to automate the construction of these hand-crafted patterns.
-By automating this process future effort could fix the reoccurring issue of scalability.
+Pattern matching is a fundamental approach that is used to seamlessly detect relationship asserting sentneces.
+In this context patterns can consist of phrases from constituency trees, a set of keywords or some combination of both to detect sentences [@doi:10.1093/nar/gkx462; @doi:10.1186/s12859-018-2103-8; @doi:10.1371/journal.pone.0152725; @doi:10.1093/bioinformatics/btg449; @doi:10.1186/1471-2105-14-181; @doi:10.1109/TCBB.2014.2372765].
+Xu et al. designed a pattern matcher system to detect sentences in PubMed abstracts that indicate drug-disease treatments [@doi:10.1186/1471-2105-14-181].
+This system matched drug-disease pairs from clinicaltrails.gov to drug-disease pairs mentioned in abstracts.
+This matching process aided the authors in identifying sentences that were used to create simple patterns, such as "Drug in the treatment of Disease" [@doi:10.1186/1471-2105-14-181], to match sentences in a wide variety of abstracts.
+Through this approach the authors achieved a high precision score of 0.904 and a low recall score of 0.131 [@doi:10.1186/1471-2105-14-181].
+This low recall score was a result of construct patterns only able to capture frequently occuring drug-disease pairs.
```

Comment:
Why is this? Is one occurrence not enough because of some rules or something fundamental that I'm missing?